GunShipAlpha/src/Game.java

```diff
@@ -212,6 +223,18 @@ public void paint(Graphics g) {
         for (ExplosionParticle explosion : ongoingExplosions) {
             explosion.draw(g);
         }
+        drawScore(g);
+        drawWaveNumber(g);
+    }
+    private void drawScore(Graphics g) {
+        g.setColor(Color.BLACK);
+        g.setFont(new Font("Arial", Font.BOLD, 24));
+        g.drawString("Score: " + score, 10, 24);
+    }
+    private void drawWaveNumber(Graphics g) {
+        g.setColor(Color.BLACK);
+        g.setFont(new Font("Arial", Font.BOLD, 24));
```

Comment:
let's remove the obvious duplication between drawScore and drawWaveNumber, eh? remember that it is going to cause me distress having to call this out rather than having you notice the duplication yourself and just not put it in, eh?